distribution/docs/src/main/resources/_contents/_securing/managing-certificates-contents.adoc

```diff
@@ -0,0 +1,351 @@
+
+[source]
+----
+keytool -importkeystore -srckeystore [MY_FILE.p12] -srcstoretype pkcs12
+ -srcalias [ALIAS_SRC] -destkeystore [MY_KEYSTORE.jks]
+ -deststoretype jks -deststorepass [PASSWORD_JKS] -destalias [ALIAS_DEST]
+----
+. The command prompts for two passwords:
+.. Input keystore passphrase is the passphrase that is used to protect cert.p12
+.. Output keystore passphrase is the passphrase that is set for the new java keystore serverKeystore.jks
+. It is recommended that the private key password be the same as the keystore password due to limitations in java.
+.. Run the following command to determine the alias name of the added current entry. It is listed after "Alias Name:"
+
+[source]
+----
+keytool -list -v -keystore serverKeystore.jks
+----
+.. Clone the existing key using the java keytool executable, filling in <CurrentAlias>, <NewAlias>, serverKeystore.jks, and password with the correct names.
+
+[source]
+----
+keytool -keyclone -alias "<CurrentAlias>" -dest "<NewAlias>" -keystore serverKeystore.jks -storepass password
+----
+.. When prompted for a password, use the same password used when the keystore was created.
+.. Delete the original alias
+
+[source]
+----
+keytool -delete -alias "<CurrentAlias>" -keystore serverKeystore.jks -storepass password
+----
+
+[NOTE]
+====
+After the keystore is successfully created, delete the jetty files used to perform the import.
+====
+
+====== Creating a Truststore
+
+[WARNING]
+====
+This walk-through details how to import a .cer certificate
+====
+
+["lowerroman"]
+. Import the certificate into a java keystore as a trusted ca certificate
+
+[source]
+----
+keytool -import -trustcacerts -alias "Trusted Cert" -file trustcert.cer -keystore serverTruststore.jks
+----
+. Enter in a keystore password when prompted.
+
+*Adding a certificate to an existing Keystore*
+
+["lowerroman"]
+. Import the certificate into a java keystore as a certificate.
+
+[source]
+----
+keytool -importcert -file newcert.cer -keystore serverKeystore.jks -alias "New Alias"
+----
+. Enter in the keystore password if prompted.
+
+====== Default Certificates
+
+${branding} comes with a default keystore that contains certificates.
+This allows the distribution to be unzipped and run immediately in a secure manner.
+If the installer was used to install the ${branding} and a hostname other than "localhost" was given, the user will be prompted to upload new trust/key stores.
+If the hostname was left as `localhost` or the hostname was changed after installation, in order to access the ${branding} instance from another machine over HTTPS (now the default for many services) the default certificates need to be replaced with a certificates that use the fully qualified hostname of the server running the ${branding} instance.
+
+The keystore is used for different services and the certificate contained within it is aliased to `localhost`.
+
+====== Demo Certificate Authority (CA)
```

Comment:
The last sentence is unclear. Where/what should the link for more information lead?